elasticsearch-api/utils/thor/templates/method.erb

```diff
@@ -30,6 +30,7 @@ module Elasticsearch
       <%= '  '*(@namespace_depth+3) + "raise ArgumentError, \"Required argument '#{required}' missing\" unless arguments[:#{required}]" + "\n" -%>
     <%- end -%>
     <%- end -%>
+    headers = arguments.delete(:header) || {}
```

Comment:
@picandocodigo Maybe `:headers` ?

Reply:
Ah, good catch! Yeah, that should be `:headers` :+1: